src/routes/auth.js

```diff
@@ -102,4 +109,39 @@ router.post(
 	})
 )
 
+router.post(
+	'/token',
+	isAuthenticated(),
+	validateSchema(authTokenSchema),
+
+	catchAsync(async (req, res) => {
+		const { email ,  oldRefreshToken : refreshToken } = req.body
```

Comment:
`oldRefreshToken` is not the parameter name @beblicarl

Reply:
i don't get this statement please?

Reply:
thank you